pxp/request_response.md

```diff
@@ -117,16 +117,46 @@ messages.
 
 ##### Blocking Response
 
-The results of the requested action should be included in the *results* object;
-its schema is action-specific. *transaction_id* is the other required entry.
+The output of the requested action (results on stdout, errors on stderr, and the
+exit code) should be included in the *output* object.
+
+Note that there's no requirement for the action's *stdout* type, as its format
+is action-specific. As for the agent operation, the *stdout* JSON schema should
+be enforced by the agent in order to guarantee the specific format expected by
```

Comment:
You're correct; we do that for pxp-agent. But I'd rather put implementation details here.

The idea to convey is that there's a contract between agent and controller about the interface of a given 'module action' pair. And the agent enforces such contract by validating the action's output; in case the output not valid, the agent will not be replying with a blocking/non-blocking response, but it will be using an RPC error (note, regardless of the exitcode).